src/View/Slot.vala

```diff
@@ -393,21 +393,13 @@ namespace Marlin.View {
             }
         }
 
-        public override void cancel () {
+        public override void close () {
             cancel_timeouts ();
 
             if (directory != null) {
```

Comment:
This condition is now checked twice in the same `close` method: https://github.com/elementary/files/blob/be976c8b7fe63d354496427fc82d7a99aacc0edc/src/View/Slot.vala#L399